src/tests/verification_real_world/update_valid_ee_certs.rs

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env cargo +nightly -Z script
```

Comment:
Nice! I've been meaning to experiment with cargo scripts but haven't had a chance. This seems like an excellent use-case.

I'm having some trouble running this, and I'm not sure if it's my fault or not. Do you have any ideas?

```
$> ./update_valid_ee_certs.rs
error: toolchain 'nightly -Z script' is not installed
```

```
$ rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/daniel/.rustup

installed toolchains
--------------------

nightly-x86_64-unknown-linux-gnu (default)
1.60-x86_64-unknown-linux-gnu
1.72-x86_64-unknown-linux-gnu

installed targets for active toolchain
--------------------------------------

aarch64-linux-android
i686-unknown-linux-gnu
wasm32-unknown-unknown
x86_64-linux-android
x86_64-unknown-linux-gnu

active toolchain
----------------

nightly-x86_64-unknown-linux-gnu (default)
rustc 1.77.0-nightly (3d0e6bed6 2023-12-21)
```

Reply:
I suspect b74940aba1bf6dd723ba651b87302931cf0b30ce fixes it. My macOS did not need this change, and I guess Linux does.

Reply:
That did the trick, thanks :+1: